src/Main.hs

```diff
@@ -34,25 +35,26 @@ checkBaseProgram baseTime baseMetric = if baseTime == -1
                                                  "Autobahn cannot optimize."
                                             else putStr $ "Base measurement is: " ++ (show baseTime)
 
-fitness :: FilePath -> String -> Double -> Int64 -> MetricType -> [FilePath] -> [BangVec] -> IO Time
-fitness projDir args timeLimit reps metric files bangVecs = do
+--fitness :: FilePath -> String -> Double -> Int64 -> MetricType -> [FilePath] -> [BangVec] -> IO Time
+--fitness projDir args timeLimit reps metric files bangVecs = do
+fitness cfg reps files bangVecs = do
```

Comment:
Good call replacing most of the args with Cfg.

Add the type of this function.